test/blackbox-tests/test-cases/github759.t/run.t

```diff
@@ -10,7 +10,7 @@
     $TESTCASE_ROOT/_build/default/.foo.objs/byte)
    (S
     $TESTCASE_ROOT)
-   (FLG (-open Foo -w -40)))
+   (FLG (-w -40)))
```

Comment:
some merlin tests got their `-open ` flag back. Why is it not the case here?

Reply:
Good eye. It's actually a ninja bug fix here. The `-open` was never needed here since this a single module library.